test/index.js

```diff
@@ -34,43 +34,34 @@ test('throws when passed non-object number using [strict: true]', ({ end, throws
   end()
 })
 
-test('returns original value when passed non-object boolean using [strict: false]', ({ end, is, doesNotThrow }) => {
-  const redact = fastRedact({ paths: ['a.b.c'], strict: false })
-  doesNotThrow(() => redact(true))
-  const o = redact(true)
-  is(o, true)
-  end()
-})
-
-test('returns original value when passed non-object null using [strict: false]', ({ end, is, doesNotThrow }) => {
-  const redact = fastRedact({ paths: ['a.b.c'], strict: false })
-  doesNotThrow(() => redact(null))
-  const o = redact(null)
-  is(o, null)
-  end()
-})
-
-test('returns original value when passed non-object undefined using [strict: false]', ({ end, is, doesNotThrow }) => {
-  const redact = fastRedact({ paths: ['a.b.c'], strict: false })
-  doesNotThrow(() => redact())
-  const o = redact()
-  is(o, undefined)
-  end()
-})
+test('returns serialized, original value when passed non-object using [strict: false]', ({ end, is, doesNotThrow }) => {
+  const customSerialize = (v) => `Hello ${v}!`
+  const redactDefaultSerialize = fastRedact({ paths: ['a.b.c'], strict: false })
+  const redactCustomSerialize = fastRedact({
+    paths: ['a.b.c'],
+    strict: false,
+    serialize: customSerialize
+  })
```

Comment:
Can you please split this test into two, depending if there is a serialize or not?